### ISSUE TITLE
[MD-UV380] Fw v0.3.7 can't connect to WPSD

### ISSUE THREAD

**Issue by rolf-tx** [OpenRTX/OpenRTX#277]
The UV380 with fw v0.3.7 is unable to connect to WPSD hotspot, no response.
The UV380 can connect to WPSD fine with v0.3.6.

The WPSD behaves like when no valid call sign has been set.
I get the same behavior using v0.3.6.0 if I have a blank call sign.

Module17 can decode the UV380 v0.3.7 ok and looks like it receiving my call sign.
UV380 v0.3.7 can decode WSPD ok.

**Comment by silseva** [comment]
Hi!
It seems that something got messed up between the two releases, then. I didn't get the bug because, unfortunately, I don't have an hotspot: all the tests I did were between radios and they worked fine, just like in your case between the radio and the Module17. I'll go through the commits to see if I can get any clue.

**Comment by marcoSchr** [comment]
I was able to reproduce this. I bisected the issue to one of two commits:

The first bad commit could be any of:
e53e110a3b8d71355ded8e6af5a6622c07da76b0
8842b98a5243b79f9527439e13ad142e5c45c3fa

**Comment by marcoSchr** [comment]
The RT3/MD380 has the same behavior

**Comment by silseva** [comment]
> I was able to reproduce this. I see no log output for the MMDVM log on wpsd. The LED for M17 illuminates, but flickers on the MMDVM.
> 
The flickering led may be the symptom of a bad transmission. I'll check with the spectrum analyser.
Can you set the MMDVM log level to verbose/debug and see if it says something, please?

**Comment by marcoSchr** [comment]
Here are the MMDVMHost logs.
Once for the failing commit and once for the working commit before.

<details> 
  <summary>8842b98a5243b79f9527439e13ad142e5c45c3fa</summary>

```
D: 2024-07-16 10:02:53.530 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:53.570 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:53.610 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:53.650 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:53.690 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:53.775 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:53.875 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:53.915 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:53.955 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:53.995 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.035 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.118 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.157 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.197 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.237 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.277 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.317 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.397 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.437 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.477 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.517 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.564 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.603 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.643 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.683 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.723 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.805 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.845 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.885 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:54.996 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.036 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.076 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.116 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.156 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.196 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.269 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.309 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.350 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.432 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.472 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.512 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.599 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.639 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.679 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.802 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.847 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.882 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.922 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:55.962 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.002 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.042 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.082 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.132 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.172 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.212 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.252 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.292 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.332 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.400 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.440 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.480 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.520 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.560 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.605 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.682 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.722 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.762 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.802 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.842 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.882 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.922 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:56.962 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:02:57.001 M17, raw RSSI: 47, reported RSSI: -47 dBm
```
</details>

<details> 
  <summary>695210d6895ad05a5a6eb289fd3b664209dbf0f8</summary>

```
D: 2024-07-16 10:03:04.953 M17, raw RSSI: 100, reported RSSI: -100 dBm
D: 2024-07-16 10:03:04.993 M17, raw RSSI: 98, reported RSSI: -98 dBm
D: 2024-07-16 10:03:05.033 M17, raw RSSI: 96, reported RSSI: -96 dBm
D: 2024-07-16 10:04:14.326 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.366 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.406 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.446 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.486 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.526 M17, raw RSSI: 47, reported RSSI: -47 dBm
M: 2024-07-16 10:04:14.526 M17, received RF late entry voice transmission from DM4RCO to ALL
D: 2024-07-16 10:04:14.699 M17, audio: FN: 24, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:14.700 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.700 M17, audio: FN: 25, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:14.705 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.706 M17, audio: FN: 26, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:14.706 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.706 M17, audio: FN: 27, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:14.712 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.712 M17, audio: FN: 28, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:14.726 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.726 M17, LSF text data fragment
D: 2024-07-16 10:04:14.727 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:14.727 M17, audio: FN: 29, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:14.766 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.767 M17, audio: FN: 30, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:14.806 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.807 M17, audio: FN: 31, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:14.846 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.847 M17, audio: FN: 32, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:14.886 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.886 M17, audio: FN: 33, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:14.926 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.927 M17, audio: FN: 34, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:14.966 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:14.966 M17, LSF text data fragment
D: 2024-07-16 10:04:14.966 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:14.967 M17, audio: FN: 35, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.006 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.006 M17, audio: FN: 36, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.046 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.046 M17, audio: FN: 37, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.086 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.086 M17, audio: FN: 38, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.126 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.126 M17, audio: FN: 39, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.166 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.166 M17, audio: FN: 40, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.206 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.206 M17, LSF text data fragment
D: 2024-07-16 10:04:15.206 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:15.207 M17, audio: FN: 41, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.246 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.247 M17, audio: FN: 42, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.286 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.286 M17, audio: FN: 43, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.326 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.326 M17, audio: FN: 44, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.366 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.366 M17, audio: FN: 45, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.406 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.406 M17, audio: FN: 46, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.446 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.446 M17, LSF text data fragment
D: 2024-07-16 10:04:15.446 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:15.447 M17, audio: FN: 47, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.486 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.486 M17, audio: FN: 48, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.526 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.526 M17, audio: FN: 49, errs: 1/272 (0.4%)
D: 2024-07-16 10:04:15.566 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.566 M17, audio: FN: 50, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.606 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.606 M17, audio: FN: 51, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.646 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.646 M17, audio: FN: 52, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.686 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.686 M17, LSF text data fragment
D: 2024-07-16 10:04:15.686 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:15.687 M17, audio: FN: 53, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.726 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.726 M17, audio: FN: 54, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.766 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.767 M17, audio: FN: 55, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.806 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.807 M17, audio: FN: 56, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.846 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.847 M17, audio: FN: 57, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.886 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.887 M17, audio: FN: 58, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.926 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.926 M17, LSF text data fragment
D: 2024-07-16 10:04:15.927 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:15.927 M17, audio: FN: 59, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:15.966 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:15.966 M17, audio: FN: 60, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.006 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.007 M17, audio: FN: 61, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.046 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.047 M17, audio: FN: 62, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.086 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.087 M17, audio: FN: 63, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.126 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.127 M17, audio: FN: 64, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.166 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.166 M17, LSF text data fragment
D: 2024-07-16 10:04:16.167 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:16.167 M17, audio: FN: 65, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.206 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.206 M17, audio: FN: 66, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.246 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.246 M17, audio: FN: 67, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.286 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.286 M17, audio: FN: 68, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.326 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.326 M17, audio: FN: 69, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.366 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.367 M17, audio: FN: 70, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.406 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.406 M17, LSF text data fragment
D: 2024-07-16 10:04:16.406 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:16.407 M17, audio: FN: 71, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.446 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.446 M17, audio: FN: 72, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.486 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.487 M17, audio: FN: 73, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.526 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.526 M17, audio: FN: 74, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.566 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.567 M17, audio: FN: 75, errs: 1/272 (0.4%)
D: 2024-07-16 10:04:16.606 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.606 M17, audio: FN: 76, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.646 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.646 M17, LSF text data fragment
D: 2024-07-16 10:04:16.647 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:16.647 M17, audio: FN: 77, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.686 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.686 M17, audio: FN: 78, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.731 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.732 M17, audio: FN: 79, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.766 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.766 M17, audio: FN: 80, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.806 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.806 M17, audio: FN: 81, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.846 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.847 M17, audio: FN: 82, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.886 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.886 M17, LSF text data fragment
D: 2024-07-16 10:04:16.886 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:16.887 M17, audio: FN: 83, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.926 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.927 M17, audio: FN: 84, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:16.966 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:16.966 M17, audio: FN: 85, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:17.006 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:17.006 M17, audio: FN: 86, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:17.046 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:17.046 M17, audio: FN: 87, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:17.086 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:17.086 M17, audio: FN: 88, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:17.126 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:17.126 M17, LSF text data fragment
D: 2024-07-16 10:04:17.126 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:17.127 M17, audio: FN: 89, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:17.166 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:17.166 M17, audio: FN: 90, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:17.206 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:17.206 M17, audio: FN: 91, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:17.246 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:17.246 M17, audio: FN: 92, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:17.286 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:17.286 M17, audio: FN: 93, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:17.326 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:17.326 M17, audio: FN: 94, errs: 0/272 (0.0%)
D: 2024-07-16 10:04:17.366 M17, raw RSSI: 47, reported RSSI: -47 dBm
D: 2024-07-16 10:04:17.366 M17, LSF text data fragment
D: 2024-07-16 10:04:17.366 0000:  00 00 00 00 00 00 00 00 00 00 00 00 00 00          *..............*
D: 2024-07-16 10:04:17.366 M17, audio: FN: 32863, errs: 0/272 (0.0%)
M: 2024-07-16 10:04:17.372 M17, received RF end of transmission from DM4RCO to ALL, 2.9 seconds, BER: 0.0%, RSSI: -47/-47/-47 dBm
```
</details>

**Comment by silseva** [comment]
Fixed by 031eda1.
The gpio of the tone generator had its pull-up enabled due to a bug introduced during the refactoring of the gpio driver. This caused the PLL frequency to be shifted of ~2kHz every time the PTT was pressed. A new binary image will be available tomorrow once the nightly build system does its run.

**Comment by rolf-tx** [comment]
@silseva 
I compiled it from git master, confirmed working with WPSD.
Thank you for quick fix :-)
